string source
string pin
string target_item
string target_pin
geometry_msgs/Pose screw_pose
geometry_msgs/Pose hole_pose